app/models/faculty.rb

```diff
@@ -7,11 +7,10 @@ class Faculty < ApplicationRecord
   has_one :school, through: :user
   has_many :startup_feedback, dependent: :nullify
   has_many :evaluated_events,
-           class_name: 'TimelineEvent',
-           foreign_key: 'evaluator_id',
+           class_name: "TimelineEvent",
```

Comment:
This file has no other change other than re-format; can you be careful not to push files with no changes related to the issue?

Reply:
My bad; Didnt spot the change